docs/checks.md

```diff
@@ -588,3 +588,15 @@ possible.
 **Remediation steps**
 - Fix the vulnerabilities. The details of each vulnerability can be found on <https://osv.dev>.
 
+## Webhooks 
+
+Risk: `High`  (possible service can be accessed by third-party)
+
+This check validate if the webhook defined in the repository have a token configured to send as validation to make sure that is a trusted request.
```

Comment:
`validate` -> `validates`
`to send as validation...` -> `to authenticate the origin of the request`